packages/docs/docs/getting-started/quickstart.mdx

```diff
@@ -122,8 +122,8 @@ export default makeProject({
 ```
 
 When creating a project, we need to provide it with an array of scenes to
-display. In this case, we use only one scene imported from
-`src/scenes/example.tsx`.
+display. In this case, we use only one scene imported from `src/scenes/example.tsx?scene`. 
+(the `?scene` is required for TypeScript to recognize the file as a scene module.)
```

Comment:
I think we can expand it a bit more and put it in its own paragraph without the parenthesis since forgetting about the `?scene` is a common mistake:
```suggestion
Note the `?scene` at the end - it's required to transform the imported module into a proper scene. Among other things, it makes it possible to dynamically refresh the preview whenever you modify the scene. The editor will not work without it.
```